app/src/features/Mappings/CreateMapping.tsx

```diff
@@ -84,17 +79,19 @@ const CreateMapping = (): JSX.Element => {
   const classes = useStyles();
   const history = useHistory();
   const dispatch = useAppDispatch();
-  const { data: credential } = useApiCredentialsListQuery({ source: sourceId });
-  const { data: owners } = useApiOwnersListQuery({
-    credential: credential?.[0].id,
-  });
-  const owner = owners?.[0];
 
   const [activeStep, setActiveStep] = useState(0);
   const [isProfileSelected, setIsProfileSelected] = useState(false);
-  const mapping = useAppSelector(selectMappingCurrent);
-  const filters = useAppSelector(selectMappingFilters);
-  const joins = useAppSelector(selectMappingJoins);
+
+  const [mapping] = useAppSelector((state) =>
+    resourceSelectors.selectAll(state)
+  );
```

Comment:
```suggestion
  const [mapping] = useAppSelector(resourceSelectors.selectAll);
```

Reply:
I find it a bit unsafe not to check on mapping's id. 
If for some reasons `mappingSlice` has more than 1 resource then it could lead to bugs